README.rst

```diff
@@ -199,7 +199,31 @@ Step 6: Add tracing to the Lambda function
             span.set_tag("example_tag", "example_value")
 
 To review more examples and usage details, see [Jaeger Python Tracer](https://github.com/signalfx/jaeger-client-python>).
-    
+
+Propagating trace context to outgoing requests or lambda response
+------------------------------------------------------------------
+
+The library ships a helper function to inject tracing context headers into a dictionary like object.
+The function accepts two arguments. First argument must be a dictionary like object that the trace context is injected into.
+The second argument is optional and must be a OpenTracing span context. If one is not provided, the function uses the currently
+active span. Example:
+
```

Comment:
I take it you've looked at a variety of outbound http APIs and settled on "here's a dictionary, add the b3 headers to it" as the most convenient approach for the majority of cases?   If so, looks good.

Reply:
Correct. Standard library uses a dictionary for headers and so does the most popular HTTP library (requests). Also opentracing python API expects a dictionary as well so it's the most convenient.